### ISSUE TITLE
Remove 'progress' animation from preview button

### ISSUE THREAD

**Issue by taylortom** [adaptlearning/adapt_authoring#1030]
This is completely useless as has no relation to the actual build progress - it's currently just set to animate across in 30secs. Suggest we take it out, or think of a better way to do this.

See: https://github.com/adaptlearning/adapt_authoring/blob/master/frontend/src/core/editor/global/views/editorView.js#L86

**Comment by brian-learningpool** [comment]
I somewhat agree with this -- the 30 seconds is arbitrary -- and this is something that should be considered as part of the UI refresh.   We do need some visual feedback that a long running process is running.

It could be replaced by a Font Awesome spinner or something though.

**Comment by taylortom** [comment]
I agree. One idea I had was some kind of status bar like apple's mail:
![apple mail status bar](https://cloud.githubusercontent.com/assets/1059083/13499418/43fdb80a-e156-11e5-892a-6bed68f95e51.png)

Note that we do still have the loading spinner on the button itself.

**Comment by taylortom** [comment]
Animation has been replaced. Have pushed this to later release to discuss the status bar idea.

**Comment by taylortom** [comment]
Another possible option is a scrolling button animation like this:

![screen shot 2016-07-12 at 13 05 04](https://cloud.githubusercontent.com/assets/1059083/16766334/6e4145f4-4831-11e6-8b0d-6c83c6ce9043.png)
